config/datasets/user-activity.json

```diff
@@ -13,16 +13,15 @@
     "options": {
         "title": "User Activity",
         "description": [
-            "Usage Behavior is a weekly report describing ways in which desktop users are interacting with the web.",
-            "**Due to changes in our data infrastructure, these charts do not show data after mid-December 2019. We are working to address this issue as soon as possible.**"
+            "Usage Behavior is a weekly report describing ways in which desktop users are interacting with the web."
```

Comment:
There's no way I would have expected you to know this, but since this is only one paragraph, we can make this a string value rather than an array:

```javascript
"description": "Usage Behavior is a weekly report describing ways in which desktop users are interacting with the web."
```

Reply:
+1